src/styles/variables.ts

```diff
@@ -176,6 +176,7 @@ export default {
     reportActionItemImagesMoreCornerTriangleWidth: 40,
     bankCardWidth: 40,
     bankCardHeight: 26,
+    popoverzIndex: 10000,
```

Comment:
Just want to make sure:
Have you compared this value to other `zIndex` values? For example `invisibleOverlay` uses `1000` and `popoverzIndex` uses `10050`.

Reply:
Yes, we need to set high zIndex value to make popover in front of attachment modal (without setting any overlay around it)